### ISSUE TITLE
Wrong plan.edit permission in route definitions

### ISSUE THREAD

**Issue by paul121** [mstenta/farm_nfa#49]
Was referencing the 2.x plan module for something.. I don't think `plan.edit` is a valid `_entity_access` value. It should be `plan.update`. It still works for the admin users, but a `farm_manager` will not have access.

https://github.com/mstenta/farm_nfa/blob/2.x/web/modules/custom/farm_nfa/farm_nfa.routing.yml#L8

**Comment by pcambra** [comment]
This was a very instructive read, as I was looking at why this might be the case, on Drupal\entity\checkEntityPermissions

```
  /**
   * Checks the entity operation and bundle permissions.
   *
   * @param \Drupal\Core\Entity\EntityInterface $entity
   *   The entity for which to check access.
   * @param string $operation
   *   The entity operation. Usually one of 'view', 'view label', 'update',
   *   'duplicate' or 'delete'.
   * @param \Drupal\Core\Session\AccountInterface $account
   *   The user for which to check access.
   *
   * @return \Drupal\Core\Access\AccessResultInterface
   *   The access result.
   */
  protected function checkEntityPermissions(EntityInterface $entity, $operation, AccountInterface $account) {
    $permissions = [
      "$operation {$entity->getEntityTypeId()}",
      "$operation {$entity->bundle()} {$entity->getEntityTypeId()}",
    ];

    return AccessResult::allowedIfHasPermissions($account, $permissions, 'OR');
  }
```

So it's basically whatever we define on the permission provider, which by default is create, update, delete, view :)\

Thanks for letting me know @paul121 !

**Comment by pcambra** [comment]
Pushed a fix for this on #48 /c @mstenta

**Comment by paul121** [comment]
@pcambra ah cool! So when we create custom permissions the first word becomes the "operation". That could be useful! Good to know.
